src/main/java/squeek/applecore/mixinplugin/MixinPlugin.java

```diff
@@ -64,8 +64,9 @@ public void acceptTargets(Set<String> myTargets, Set<String> otherTargets) {
                 LOG.info("Found " + mod.modName + "! Integrating now...");
             }
             else {
-                LOG.error("Could not find " + mod.modName + "! Going to crash now");
-                FMLCommonHandler.instance().exitJava(-99, false);
+                LOG.error("Could not find " + mod.modName + "! Ought to crash now");
+                if (!isDevelopmentEnvironment)
+                    FMLCommonHandler.instance().exitJava(-99, false);
```

Comment:
Why is there a close on crash anyway? It is simply supposed to skip a mod if not loaded. Unless it is required. But that should be handled somewhere else!

Reply:
This file is based on this one: https://github.com/SinTh0r4s/ExampleMod1.7.10/blob/4fe186c4d8bb7bdcab1f81fede9de92ae350d397/src/main/java/com/myname/mymodid/mixinplugin/MixinPlugin.java#L66

No need to kill a run if a mod is not found!

Reply:
Due to renaming shenanigans it just skipped applying the mixins; this will ensure that they're applied when not in dev!

Reply:
I cannot remember to change the jar name of Pam's. What changed there? In that case you may extend the matching of mods? Or adapt the input to the Mixins module? I think to fail Mixins is the wrong place to do it. I doesn't seem to be the module itself, but it's configuration.

Reply:
The problem is we run in production against our copy of Pam's but that is hidden source, so it's not on jitpack. As a workaround we "compileOnly" against Pams from Curse, but then if you runClient it can't find Pam's (and the mixin apply then fails).

To prevent the mixin apply from failing in production, we error out if the mixin cannot apply (against the renamed Pams which was renamed because Technic can't have a space in the filename there for something we supply). This puts a "dev" wrapper around that check so that we don't error out in a runClient (dev) enviroment.

In this case a missing mod is an error condition and we should packExit (and thereby catch it before shipping when we have what we had in 2.1.2.0 which was a mixin failing to apply).

Alternatively, I can add a config wrapping the check and then set the config false by default but true in the pack (similar to the RAM check added and others).